styles/style.css

```diff
@@ -248,3 +248,20 @@ p {
     padding-right: 2rem;
   }
 }
+
+button {
+  background-color: var(--white);
+  border: 1px solid transparent;
+  border-radius: 8px;
+  padding: 0.5rem;
+  color: var(--orange-dark);
+  font-weight: bolder;
+}
```

Comment:
It would be a good practice to define the appearance of the Volunteer button with a **new css class**, instead of defining it only on button element.

This can ensure the definition do not affect other button elements, such as  `.brownbtn`, and reduce conflicts when you have more types of buttons.

To improve, we can extract similarity across buttons and generalise it on the button element. For example:

```
button {
   border: 1px solid transparent;
   border-radius: 8px;
   padding: 0.5rem;
   font-weight: bolder;
}
.whitebtn {
  background-color: var(--white);
  color: var(--orange-dark);
}
```

Reply:
wow, this is the best idea, thank you for suggestion